README.md

```diff
@@ -262,6 +262,9 @@ environment variable:
   this pointer. Note that it is not currently guaranteed that code that works
   with `-Zmiri-track-raw-pointers` also works without
   `-Zmiri-track-raw-pointers`, but for the vast majority of code, this will be the case.
+* `-Zmiri-panic-on-unsupported` will makes some forms of unsupported functionality
+  such as FFI and unsupported syscalls panic within the context of the emulated
+  application instead of raising an error within the context of Miri.
```

Comment:
Please keep this list alphabetically sorted.

Reply:
Makes sense, didn't notice there was explicit ordering.